local_planner/src/nodes/box.cpp

```diff
@@ -1,41 +1,25 @@
 #include "box.h"
 
-Box::Box(double x_min, double x_max, double y_min, double y_max, double z_min,
-         double z_max)
-    : xmin_{x_min},
-      xmax_{x_max},
-      ymin_{y_min},
-      ymax_{y_max},
-      zmin_{z_min},
-      zmax_{z_max} {}
+Box::Box(const double radius)
```

Comment:
Just stumbled on this; there's little added benefit in declaring const if you're passing by value. I believe what Martina meant was that you should generally pass by constant reference any value that is larger than 32 bits